cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -90,7 +90,7 @@ credentials "auth_flexera" do
 end
 
 ###############################################################################
-# Datasources
+# Datasources & Scripts
```

Comment:
I think this should be only "Datasources" in the scripts section you can put "Scripts".

Reply:
Hi Juan regarding this article; Datasources and Scripts now are presented in the order of execution followed by any script blocks that they call.
https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices